packages/orchestration/src/exos/ibc-packet.js

```diff
@@ -128,7 +133,7 @@ export const prepareIBCTransferSender = (zone, { watch, makeIBCReplyKit }) => {
           const { resultV: ackDataV, ...rest } = makeIBCReplyKit(
             replyPacketPattern,
             match,
-            ctx,
+            opts,
```

Comment:
This is the right place to declare that the operation is a transfer, unless `opts` overrides it.

```suggestion
            { opName: 'transfer', ...opts },
```